functional_tests/steps/bundles.py

```diff
@@ -121,3 +129,337 @@ def the_user_can_see_the_bundle_details_with_creator(context: Context) -> None:
 def bundle_inspect_show(context: Context) -> None:
     expect(context.page.get_by_text("Created by")).to_be_visible()
     expect(context.page.get_by_text(context.bundle_creator.get_username())).not_to_be_visible()
+
+
+# Bundles UI Data Setup
+
+
+@given("there is a {user_role} user")
+def a_user_exists_by_role(context: Context, user_role: str) -> None:
+    if not hasattr(context, "users"):
+        context.users = {}
+    context.users[user_role] = create_user(user_role)
+
+
+@given("there is a Statistical Analysis page")
+def statistical_analysis(
+    context: Context,
+) -> None:
+    context.article_series_page = ArticleSeriesPageFactory(title="PSF")
+    article = StatisticalArticlePageFactory(parent=context.article_series_page)
+    mark_page_as_ready_to_publish(article, UserFactory())
+    context.statistical_article_page = article
+
+
+@given("there is a release calendar page")
+def release_calendar(context: Context) -> None:
+    nowish = timezone.now() + timedelta(minutes=20)
+    context.release_calendar_page = ReleaseCalendarPageFactory(
+        release_date=nowish, title="Release Calendar Page bundles UI", status=ReleaseStatus.PROVISIONAL
+    )
+
+
+@given("there is a preview team")
+def preview_teams_create(context: Context) -> None:
+    context.team = Team.objects.create(identifier="preview-team", name="Bundles UI Preview team", is_active=True)
+
+
+@given("the {user_role} is a member of the preview team")
+def add_user_to_preview_teams(context: Context, user_role: str) -> None:
+    user = context.users[user_role]["user"]
+    user.teams.add(context.team)
+
+
+@given("there are {number_of_bundles} bundles with {bundle_details}")
+def multiple_bundles_create(context: Context, number_of_bundles: str, bundle_details: str) -> None:
+    bundle_dets = json.loads(bundle_details)
+    if bundle_dets:
+        context.bundles = []
+        context.bundlepages = []
+        context.bundleteams = []
```

Comment:
For consistency can't we have:
```suggestion
        context.bundle_pages = []
        context.bundle_teams = []
```

Reply:
Second point here, why save this in `context` if you don't use it anywhere else? Am I missing something?